types/polylabel/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for polylabel 1.0
```

Comment:
There's an item in the PR checklist:

>  If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

I *think* this is what's meant?

Reply:
Yep, that's all you need to do 😊